specs/conductor.md

```diff
@@ -152,36 +149,34 @@ The `chain_id` that Conductor uses as the rollup's identifier is pulled from the
 rollup's config. When a user submits a transaction to be sequenced, they specify
 the `chain_id` of its destination.
 
-When a Sequencer block is received from the gossip network, the Conductor
+When a Sequencer block is received, the Conductor
 filters the transactions for its chain ID and executes only those transactions
 on top of its parent state. See the [astria execution
-api](https://github.com/astriaorg/astria/blob/main/specs/execution-api.md) for
-more details.
+api](https://github.com/astriaorg/astria/blob/main/specs/execution-api.md) for more details.
 
 ### Data Validation
 
 Data is validated before being sent to the rollup for execution. Validation
 occurs in two places:
 
-- When blocks are received from the gossip network, the data is validated in
+- When blocks are received directly from the sequencer, the data is validated in the
   Driver's `handle_network_event` using
   `BlockVerifier::validate_sequencer_block_data`
-- When blocks are fetched from the DA layer, the data is validated in Reader's
+- When blocks are fetched from the DA layer, the data is validated in the Reader's
   `get_new_blocks` using `BlockVerifier::validate_signed_namespace_data` and
   `BlockVerifier::validate_rollup_data`
 
 ### Soft Commitments
 
-When a Sequencer block is received by the Conductor from the gossip network, it
-is assumed to be the `head` of the chain, thus its parent block is designated as
-a `soft` commitment. This is also the primary moment in the data life cycle for
-Astria that transactions are sent to the rollup for execution.
+When a block is received by the Conductor directly from the sequencer, it
+is assumed to be valid based on CometBTF consensus and immediately set as a `soft` commitment.
```

Comment:
```suggestion
is assumed to be valid based on CometBFT consensus and immediately set as a `soft` commitment.
```

Reply:
also it isn't assumed to be valid, we should fetch the commit for it + check that as well (has not been implemented)